CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
   - Avoid basing results on old analysis.
   - Add new setting `:completion :analysis-type` to choose between `:fast-but-stale`(default) or `:slow-but-accurate`, this should define whether completion should wait for changes that may still happening, this by default reverts the behavior introduced after #1425. #1487
 
+- Editor
+  - Fix `textDocument/hover` issue when doc metadata isn't a string literal.
+  - Follow references to other vars in doc metadata for use in `textDocument/hover`.
```

Comment:
Thre is already a Editor entry [here](https://github.com/clojure-lsp/clojure-lsp/pull/1490/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR8) 😅
```suggestion
  - Fix `textDocument/hover` issue when doc metadata isn't a string literal.
  - Follow references to other vars in doc metadata for use in `textDocument/hover`.
```

Reply:
:facepalm: Thanks